src/pages/Armies/ViewArmiesPage.jsx

```diff
@@ -1,14 +1,44 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import PageTitle from '../../atoms/PageTitle';
-import CodexFilter from '../../molecules/CodexFilter';
 import RedirectButton from '../../atoms/RedirectButton';
+import DeleteButton from '../../molecules/DeleteButton';
+import store from '../../Redux/store';
 
 function ViewArmiesPage() {
```

Comment:
Shouldn't this be written in ES6 without the return?

```
const ViewArmiesPage = () => (
```

Reply:
For this website I made all my pages as functions that returned something.  Is that not correct?  For my last movie website I made all the pages as:  const Favoritespage = () => {......, return X }, so they still had returns in them.  Example:   https://github.com/cawensley/moviegallery/blob/master/src/pages/favorites.jsx

Reply:
Well, I meant writing functions the more modern way. Avoiding `function` wherever possible is usually a good idea.